### PR TITLE
[Reports Page] Display multi env info for promotions & merge_activities

### PR DESCRIPTION
**Pull request by violetyao**
Jira: https://issues.redhat.com/browse/APPSRE-2953
Example (`promotions` and `merge_activities` not null): 
Screen Shot 2021-03-22 at 1 35 13 PM

Example (`promotions` and `merge_activities` null): 
Screen Shot 2021-03-22 at 1 35 22 PM

### CODE REVIEW

File: src/pages/elements/Report.js [hunk 1]
```diff
@@ -381,12 +537,36 @@ const DeploymentValidations = ({ get_ns, deployment_validations}) => {
...
 
-function Report({ report, namespaces}) {
+function Report({ report, namespaces, saas_files}) {
   const content = yaml.safeLoad(report.content);
 
   // fetch the namespace. Returns `undefined` if not found
   const get_ns = (c, ns) => namespaces.filter(n => n['name'] === ns && n['cluster']['name'] === c)[0];
 
+  let promotionSection;
+  if (content.production_promotions) {
```
Comment (BumbleFeng) [inline src/pages/elements/Report.js:547]: switch content.production_promotions to else if and add comments for removing these when we get rid of all old reports

File: src/pages/elements/Report.js [hunk 2]
```diff
@@ -381,12 +537,36 @@ const DeploymentValidations = ({ get_ns, deployment_validations}) => {
...
+    promotionSection = <Promotions promotions={content.promotions} get_ns={get_ns} saas_files={saas_files}/>;
+  } else {
+    promotionSection = (<React.Fragment>
+                        <h4>Promotions</h4>
+                        <p style={{ 'font-style': 'italic' }}>No promotions.</p>
+                       </React.Fragment>);
+  }
+
+  let mergeSection;
+  if (content.merges_to_master) {
```
Comment (BumbleFeng) [inline src/pages/elements/Report.js:559]: same here

**Review by BumbleFeng — APPROVED** [review]
LGTM

File: src/pages/elements/Report.js (hunk 1 above)
Comment (violetyao) [reply src/pages/elements/Report.js]: Got it!

**Pull request merged**